add_up.rb

```diff
@@ -0,0 +1,14 @@
+def add_up(i)
```

Comment:
Well done. The return of sum doesn't have to be inside the if statement. Nesting it in the if statement will mean that when the if the conditions aren't met for the if statement then the range will likely be returned by default when the method completes.